project/ProjectPlugin.scala

```diff
@@ -158,37 +151,21 @@ object ProjectPlugin extends AutoPlugin {
 
   }
 
-  import autoImport._
-
   override def projectSettings: Seq[Def.Setting[_]] =
-    Seq(
-      resolvers += Resolver.bintrayRepo("beyondthelines", "maven"),
-      orgAfterCISuccessTaskListSetting := List(
-        depUpdateDependencyIssues.asRunnableItem,
-        orgPublishReleaseTask
-          .asRunnableItem(allModules = true, aggregated = false, crossScalaVersions = true),
-        orgUpdateDocFiles.asRunnableItem
-      ),
+    scalaMetaSettings ++ sharedReleaseProcess ++ warnUnusedImport ++ Seq(
+      libraryDependencies ++= commonDeps :+ %("slf4j-nop") % Test,
+      Test / fork := true,
+      Tut / scalacOptions ~= (_.filterNot(_ == "-Ywarn-unused-import")),
+      orgAfterCISuccessTaskListSetting ~= (_.filterNot(_ == defaultPublishMicrosite)),
       orgBadgeListSetting := List(
         TravisBadge.apply,
-        CodecovBadge.apply, { info =>
-          MavenCentralBadge.apply(info.copy(libName = "frees"))
-        },
+        CodecovBadge.apply,
+        { info => MavenCentralBadge.apply(info.copy(libName = "frees-rpc")) },
         ScalaLangBadge.apply,
         LicenseBadge.apply,
         // Gitter badge (owner field) can be configured with default value if we migrate it to the frees-io organization
-        { info =>
-          GitterBadge.apply(info.copy(owner = "47deg", repo = "freestyle"))
-        },
+        { info => GitterBadge.apply(info.copy(owner = "47deg", repo = "freestyle")) },
         GitHubIssuesBadge.apply
       )
-    ) ++ Seq(
-      fork in Test := true,
-      addCompilerPlugin(%%("scalameta-paradise") cross CrossVersion.full),
-      libraryDependencies ++= commonDeps ++ Seq(%%("scalameta", V.scalameta)),
-      scalacOptions ++= Seq("-Ywarn-unused-import", "-Xplugin-require:macroparadise"),
-      scalacOptions in Tut ~= (_ filterNot Set("-Ywarn-unused-import", "-Xlint").contains),
-      scalacOptions in (Compile, console) ~= (_ filterNot (_ contains "paradise")) // macroparadise plugin doesn't work in repl yet.
```

Comment:
I don't think these settings are provided by neither _sbt-org-policies_ nor _sbt-freestyle_.

Reply:
Oops, sorry, I've just seen you have moved them upper.